python/ray/scripts/scripts.py

```diff
@@ -1353,23 +1353,38 @@ def timeline(address):
     type=str,
     default=ray_constants.REDIS_DEFAULT_PASSWORD,
     help="Connect to ray with redis_password.")
+@click.option(
+    "--group-by",
+    type=click.Choice(["NODE_ADDRESS", "STACK_TRACE"]),
+    default="NODE_ADDRESS",
+    help="Group object references by a GroupByType \
+(e.g. NODE_ADDRESS or STACK_TRACE).")
+@click.option(
+    "--sort-by",
+    type=click.Choice(["PID", "OBJECT_SIZE", "REFERENCE_TYPE"]),
+    default="OBJECT_SIZE",
+    help="Sort object references in ascending order by a SortingType \
+(e.g. PID, OBJECT_SIZE, or REFERENCE_TYPE).")
 @click.option(
     "--stats-only",
-    is_flag=True,
     type=bool,
     default=False,
-    help="Connect to ray with redis_password.")
-def memory(address, redis_password, stats_only):
+    help="Display node stats store summary only.")
+@click.option(
+    "--line-wrap",
+    type=bool,
```

Comment:
Can you use store_true instead?

Reply:
oh sorry. I didn't mean to change the name of the option, but the type ()https://docs.python.org/3/library/argparse.html#allow-abbrev